src/main/java/uk/gov/digital/ho/hocs/document/DocumentDataService.java

```diff
@@ -38,9 +38,15 @@ public DocumentDataService(DocumentRepository documentRepository, S3DocumentServ
         this.documentClient = documentClient;
     }
 
-    public DocumentData createDocument(UUID externalReferenceUUID, String displayName, String fileName, String type, String convertTo) {
+    public DocumentData createDocument(
+            UUID externalReferenceUUID,
+            UUID actionDataItemUuid,
+            String displayName,
+            String fileName,
+            String type,
+            String convertTo) {
         log.debug("Creating Document: {}, external Reference  UUID: {}", displayName, externalReferenceUUID);
-        DocumentData documentData = new DocumentData(externalReferenceUUID, type, displayName);
+        DocumentData documentData = new DocumentData(externalReferenceUUID, actionDataItemUuid, type, displayName);
```

Comment:
Up to you if you change this... as we do it all over the place on DECS... but it might be better to create overloaded contructors and methods instead of the having to alter all the tests and passing `null` all over the place. I think it would make for better more maintainable code, and will stop us getting stung with null pointer issues if anything has been missed.

Reply:
It would be neater. I'll make the change.